### DISCUSSION TITLE
Organization

### DISCUSSION THREAD

**Discussion by Michael21107** [scripthookvdotnet/scripthookvdotnet#1329] [Ideas]
@kagikn why are ur repos under shvdn organization with u being the only one there?

**Comment by kagikn** [comment]
Well, organization memberships are hidden by default and individual member can select their visibility (hidden by default I guess). crosire hasn't publicized his membership. He hasn't involved in this project since he gave me an admin role for this project (and focuses on ReShade).
Me and crosire are the only members of the organization at the moment, and I keep his admin role just in case I got hit by something too bad.

**Reply by Michael21107** [reply]
Oh, didn't think it wouldnt even count hidden members of the organization, got it then, thanks
haven't you thought about renaming the organization, at least to don't 100% match svdn repo' name?

**Reply by kagikn** [reply]
Maybe later, but I'm not planning to rename the organization right now. Btw, there's organizations where the names 100% match one of their repos (in the case insensitive way), such as Prettier.

**Reply by Michael21107** [reply]
im new to GitHub so I dont know much organ., but up to you ofc
